### PR TITLE
disable model flatten for new service

### PR DESCRIPTION
**Pull request by msyyc**
https://github.com/Azure/sdk-release-request/issues/4871
https://github.com/Azure/sdk-release-request/issues/4808
https://github.com/Azure/sdk-release-request/issues/4828
https://github.com/Azure/sdk-release-request/issues/4838
https://github.com/Azure/sdk-release-request/issues/4836
https://github.com/Azure/sdk-release-request/issues/4878
https://github.com/Azure/sdk-release-request/issues/4844

### CODE REVIEW

File: specification/playwrighttesting/resource-manager/readme.md [hunk 1]
```diff
@@ -32,6 +32,11 @@ openapi-subtype: rpaas
 tag: package-2023-10-01-preview
 ```
 
+``` yaml
```
Comment (haolingdong-msft) [inline specification/playwrighttesting/resource-manager/readme.md:35]: why we don't need to update `client-flattened-annotation-target: disabled` in readme.java.md for this RP?

Comment (msyyc) [reply specification/playwrighttesting/resource-manager/readme.md:35]: It is added by @weidongxu-microsoft. And this PR is only for Python/JS/C#

Comment (weidongxu-microsoft) [reply specification/playwrighttesting/resource-manager/readme.md:35]: already in main https://github.com/Azure/azure-rest-api-specs/blob/main/specification/playwrighttesting/resource-manager/readme.java.md

**Review by haolingdong-msft — APPROVED** [review]

**Review by live1206 — APPROVED** [review]

**Review by kazrael2119 — APPROVED** [review]

**Comment by msyyc** [comment]
Go failure is expected: https://github.com/Azure/azure-rest-api-specs/pull/27367#issuecomment-1891235071

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**